packages/skeleton-react/src/composed/accordion/modules/types.ts

```diff
@@ -0,0 +1,37 @@
+import * as accordion from '@zag-js/accordion';
+import type { ButtonHTMLAttributes, HTMLAttributes } from 'react';
+import type { PropsWithChildOrChildren } from '../../../internal/props-with-child-or-children.js';
+
+interface AccordionRootProps
```

Comment:
Let's make sure we're exposing component types to resolve the recent requests for such. I believe we have a pending ticket for this too, which you might can add to to the list this change solves.